packages/select/tests/Select.test.js

```diff
@@ -734,4 +734,92 @@ describe('Select', () => {
       expect(optionOne).toHaveAttribute('name');
     });
   });
+  test('renders error message in placeholder with invalid', async () => {
+    const { container, getByText } = render(
+      <Form
+        initialValues={{
+          singleSelect: undefined,
+        }}
+        onSubmit={() => {}}
+        validationSchema={singleValueSchema('singleSelect')}
+      >
+        <Select name="singleSelect" options={options} data-testid="single-select" />
+        <Button type="submit">Submit</Button>
+      </Form>
+    );
+
+    await fireEvent.click(getByText('Submit'));
```

Comment:
I don't think we need `await` for the `fireEvent` functions. It should be safe to remove here and in the other places too